### DISCUSSION TITLE
Releases timeline

### DISCUSSION THREAD

**Discussion by vlados** [livewire/livewire#5877] [3. Feature ideas]
I think that there should be a releases timeline to make Livewire as great as possible. I am sure that there is a lot of future ideas that will come for the next major release so we should have a release date of v3 and for the next major release.
Livewire is getting implemented in bigger and bigger projects and it should be reliable as any other big framework

**Comment by mcpuishor** [comment]
Given the major changes that have been implemented in Livewire 3, I think it would be unreasonable to ask from @calebporzio and the Livewire team to commit to a timeline. 

I think I'm not the first to say that Livewire 3 has been one of the most awaited projects to be released, and I'm sure there was massive pressure to deliver. Let's give the core team some time to deal with the beta version.
